src/run_atari_ppo.py

```diff
@@ -0,0 +1,67 @@
+"""Trains PPO agent and plays Atari Pong."""
+#!/usr/bin/env python
+
+import os.path as osp
+import logging
+from mpi4py import MPI
+import gym
+from baselines.common import set_global_seeds
+from baselines import bench
+from baselines import logger
+
+def wrap_train(env):
+    """Helper function."""
+    from baselines.common.atari_wrappers import (wrap_deepmind, FrameStack)
+    env = wrap_deepmind(env, clip_rewards=True)
+    env = FrameStack(env, 4)
+    return env
+
+def train(env_id, num_frames, seed):
+    """Train agent."""
+    from baselines.ppo1 import pposgd_simple, cnn_policy
+    import baselines.common.tf_util as U
+    rank = MPI.COMM_WORLD.Get_rank()
+    sess = U.single_threaded_session()
+    sess.__enter__()
+    if rank != 0:
+        logger.set_level(logger.DISABLED)
+    workerseed = seed + 10000 * MPI.COMM_WORLD.Get_rank()
+    set_global_seeds(workerseed)
+    env = gym.make(env_id)
+    def policy_fn(name, ob_space, ac_space): #pylint: disable=W0613
```

Comment:
If the arg is unused u can denote it with `_` and get rid of the pylint disable.